Assets/Scripts/Player/Input Parser/InputParser.cs

```diff
@@ -9,12 +10,15 @@ public sealed class InputParser : MonoBehaviour
         private PlayerInput _playerInput;
         private InputActionAsset _inputActionAsset;
         private PlayerMovement _playerMovement;
+        private SwitchMenus _switchMenus;
 
         private void Awake()
         {
             _playerInput = GetComponent<PlayerInput>();
             _inputActionAsset = _playerInput.actions;
             _playerMovement = GetComponent<PlayerMovement>();
+            GameObject canvas = GameObject.Find("Canvas");
```

Comment:
2 dingen:

1. Op develop is er een `maincanvas` prefab, daar kan de pause menu in.
2. Kan je een const maken voor deze string, dat maakt het minder hardcoded.